README.md

```diff
@@ -0,0 +1,9 @@
+# Universal Packaging Tool: Macports backend
+This is a [Macports](https://www.macports.org) backend for [upt](https://pypi.python.org/pypi/upt).
+
+#### Steps to use this tool out:
+
+- Install upt 
+- Clone this repository and run ```python3 setup.py install``` to install macports backend
+- Run ```$ upt -f pypi -b macports package_name```
+- Portfile output will be printed via stdout
```

Comment:
You can leave these instructions for now. They would be updated later when the packages are added (chicken-and-egg problem, leave it as is for now).